md_api/src/main/java/com/wipro/ats/bdre/md/api/RegisterFile.java

```diff
@@ -22,19 +22,16 @@
 import org.apache.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.config.AutowireCapableBeanFactory;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.support.ClassPathXmlApplicationContext;
-import org.springframework.stereotype.Component;
 
 import java.sql.Timestamp;
 
 /**
  * Created by arijit on 12/8/14.
  */
-@Component
 public class RegisterFile extends MetadataAPIBase {
     public RegisterFile() {
-
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
     }
 
     private static final Logger LOGGER = Logger.getLogger(RegisterFile.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)